### ISSUE TITLE
Modify LMB -> OWL Jython code to run from KB

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/pipeline#1]
Dedicated branch for this here:

https://github.com/VirtualFlyBrain/VFB_owl/tree/owl_from_neo

Progress:

Conversion works for datasets associated with DBs (e.g. flyCircuit).  Definition rolling works, as does adding voxel overlap for FlyCircuit neurons.

To fix:
- [x] For datasets not assoc with DB to work, either KB schema or neo queries in script (vfb_ind_tools.py:115) need work for it to work for other datasets.  This should be pretty trivial to fix.

- [x] Update to function to add neurons to cluster mapping in OWL is incomplete  (probably should be subticket):
    -  add_clusters seems to now have code for adding cluster -> Ind links.  Logic of this should be checked.
   -  Neo query seems to be invalid.  Looks like it should be reasonably quick to fix.

Estimate 0.5-1 days work.

**Comment by dosumis** [comment]
New issue.  Getting weird database errors for 2 datasets - Jenett2012 & Chiang2010:

Unable to load NODE with id -1.... 

This doesn't look like something I can fix in code.  Looks like a DB issue...  This might be relevant: https://github.com/neo4j/neo4j/issues/7546

This error happens consistently with these datasets when running from code. BUT - testing queries directly through http://kb.virtualflybrain.org/browser/ works fine.

Now seems to be fixed.  Not sure how/why.  Leaving this comment as a record in case problem recurs in future.

**Comment by dosumis** [comment]
QA tests show clusters are either not generated or are being dropped somewhere when full build job is run.  Local tests of Chiang + clusters build worked fine though so shouldn't be too hard to fix.

**Comment by dosumis** [comment]
Seems to be failure to merge back in flycircuit_plus once building is complete (which makes me wonder how this has been working up to now!)  Fixed in VirtualFlyBrain/VFB_owl#448308033503d80075bb8e539e55532de6ab15f6
